File: test_hmac.js
const crypto = require('crypto');
const timestamp = "1740614000000";
const secret = "super_secret_community_key_2026";
const payload = `request-rembg-v1:${timestamp}`;
const sig = crypto.createHmac('sha256', secret).update(payload).digest('hex');
console.log(`JS sig: ${sig}`);

File: test_hmac.py
import time
import hmac
import hashlib

timestamp = "1740614000000"
secret = "super_secret_community_key_2026"
payload = f"request-rembg-v1:{timestamp}"
expected_sig = hmac.new(secret.encode('utf-8'), payload.encode('utf-8'), hashlib.sha256).hexdigest()
print(f"Python sig: {expected_sig}")
